src/stores/useSlices.ts

```
@@ -0,0 +1,10 @@
+import { StateCreator } from 'zustand';
+
+// zustand store slice template.
+
+// eslint-disable-next-line @typescript-eslint/no-empty-object-type
+export interface Slice {}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)